isNode.js

```diff
@@ -0,0 +1,2 @@
+const isNode = require('detect-node');
+module.exports = isNode;
```

Comment:
super simply handy dandy thingy I figure we might as well expose it to people in an universal way.  Both client and server can do 

`imports isNode from 'boringbits/isNode'`  then immediately have a `boolean` to check